fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/ReindexResourceDAO.java

```diff
@@ -294,7 +294,9 @@ public void updateParameters(String tablePrefix, List<ExtractedParameterValue> p
 
         if (parameters != null) {
             JDBCIdentityCache identityCache = new JDBCIdentityCacheImpl(getCache(), this, parameterDao, getResourceReferenceDAO());
-            try (ParameterVisitorBatchDAO pvd = new ParameterVisitorBatchDAO(connection, null, tablePrefix, false, logicalResourceId, 100,
+            // Check if this is multitenant
+            boolean mtId = this.getFlavor().isMultitenant();
```

Comment:
did this part already get committed as part of #2115 ?

Reply:
Yes, this is just an artifact of the merge (I solved the issue in this PR, and ended up having to merge)

Reply:
maybe rebase onto main and see if this gets cleaned up?

Reply:
resolved, I checked out the file from the main branch and replaced directly